azext_edge/edge/providers/support/dataprocessor.py

```diff
@@ -27,7 +27,6 @@
     'nats',
     'aio-dp-runner-worker',
     'aio-dp-operator',
-    'nfs-server-provisioner'
 ]
 
 DATA_PROCESSOR_PREFIX = "aio-dp-"
```

Comment:
Are unused label variables going to be removed ? 

Like `DATA_PROCESSOR_ONEOFF_LABEL`, `DATA_PROCESSOR_PART_OF_LABEL`, `DATA_PROCESSOR_INSTANCE_LABEL ` etc ?

Reply:
also don't forget that conftest may have some now unused stuff